cfme/infrastructure/config_management.py

```diff
@@ -1,28 +1,29 @@
 from functools import partial
+from navmazing import NavigateToSibling, NavigateToAttribute
 from cached_property import cached_property
+
 import cfme
 import cfme.fixtures.pytest_selenium as sel
 import cfme.web_ui.flash as flash
-from cfme.web_ui.menu import nav
 import cfme.web_ui.tabstrip as tabs
 import cfme.web_ui.toolbar as tb
 from cfme.web_ui import (
-    accordion, Quadicon, Form, Input, fill, form_buttons, mixins, SplitTable, Table, Region,
-    AngularSelect, Select
+    accordion, Quadicon, Form, Input, fill, form_buttons, mixins, Table, Region,
+    AngularSelect, summary_title
 )
 from utils import version, conf
 from utils.log import logger
 from utils.pretty import Pretty
 from utils.update import Updateable
 from utils.wait import wait_for
+from utils.appliance.endpoints.ui import navigator, CFMENavigateStep, navigate_to
```

Comment:
http://cfme-tests.readthedocs.io/guides/dev_guide.html

not in pep, just cfme read-the-docs